joerd/command.py

```diff
@@ -173,6 +207,98 @@ def joerd_process(cfg):
     j.process()
 
 
+def joerd_server(global_cfg):
+    assert cfg.sqs_queue_name is not None, \
+        "Could not find SQS queue name in config, but this must be configured."
+
+    sqs = boto3.resource('sqs')
+    queue = sqs.Queue(cfg.sqs_queue_name)
+
+    while True:
+        for message in queue.get_messages():
+            region = json.loads(message.body)
+            cfg = global_cfg.copy_with_regions([region])
+            joerd_process(cfg)
```

Comment:
Do we want to think about a way to shut these down? Or if we can tell when we're all done, do we just shut down the instances in a different way?